src/kOS/Module/kOSVesselModule.cs

```diff
@@ -166,6 +166,7 @@ public void FixedUpdate()
                     ClearParts();
                     HarvestParts();
                     partCount = Vessel.Parts.Count;
+                    ResetPhysicallyDetachedParameters();
```

Comment:
This seemed like the right place to put it, as this is the stuff where it checks for when the vessel adds or removes parts due to staging, docking, etc.